app/components/employeeCard.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styles from "./employeeCard.module.scss";
+
+export default function EmployeeCard(props: {
+  employeeName: string;
+  employeeRole: string;
+  employeeImageSrc: string;
+}) {
+  return (
+    <div className={styles["employeeCard"]}>
+      <div>
+        <h2>{props.employeeName}</h2>
```

Comment:
I don't know the design, but I'm not sure of the benefit of using h2 here in terms of rotor navigation with screen readers. Would love some insight on this! 

Off the bat I'd argue that this should be a `<p/>` styled as an h2 as to avoid a list of x names in the rotor menu. 🤔